CMakeLists.txt

```diff
@@ -173,7 +173,7 @@ ExternalProject_Add(${TBB_TARGET}
        BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/tbb
        INSTALL_DIR ${CMAKE_INSTALL_PREFIX}
        CONFIGURE_COMMAND ""
-       BUILD_COMMAND PREFIX=<INSTALL_DIR> make PREFIX=<INSTALL_DIR>
+       BUILD_COMMAND PREFIX=<INSTALL_DIR> make PREFIX=<INSTALL_DIR> -j$(nproc)
```

Comment:
Are we sure that nproc cannot be empty?
If it's empty it will do "-j" which could overload the machine.

Reply:
the command `nproc` should always return a value afaik.